aten/src/ATen/native/mps/operations/Shape.mm

```diff
@@ -417,7 +402,7 @@ void check_shape_except_dim(const Tensor &first, const Tensor &second,
           MPSGraphTensor* outputTensor = [mpsGraph concatTensors:inputTensorsArray
                                                        dimension:dimension // Maybe convert this from int64_t -> int32
                                                             name:nil];
-
+          // FIXME: this is leaking.
```

Comment:
Can we fix this ?

Reply:
I want to post a separate PR for that as it has nothing to do with Upsample.

Reply:
Updated the PR.